File: Front-end/app/common/service/testTask.service.js
export default function testTaskService($http) {
    var service = this;

    service.getTestTasksByEmployeeId = function (employeeId) {
        return $http({
            method: "GET",
            url: EM_REST_SERVER_URI + "/employees/" + employeeId + "/tasks"
        }).then(function (response) {
            return response.data;
        });
    };

    service.addTestTask = function (employeeId, testId) {
        return $http({
            method: "POST",
            url: EM_REST_SERVER_URI + "/employees/" + employeeId + "/tasks",
            data: {
                testId: testId
            }
        }).then(function (response) {
            return response.data;
        });
    };
}

File: Back-end/src/main/java/by/bsuir/em/service/TestTaskService.java
package by.bsuir.em.service;

import by.bsuir.em.dto.TestTaskDto;

import java.util.List;

public interface TestTaskService {
    List<TestTaskDto> getTestTasksByEmployeeId(Long employeeId);
    TestTaskDto addTestTask(TestTaskDto testTaskDto);
    void deleteTestTask(Long employeeId, Long testId);
}
